fe1-web/model/objects/HDWallet.ts

```diff
@@ -0,0 +1,127 @@
+import { WalletStore } from 'store/stores/WalletStore';
+import { WalletCryptographyHandler } from './WalletCryptographyHandler';
+
+/**
+ * bip39 library used for seed generation and verification
+ * https://www.npmjs.com/package/bip39
+ */
+const bip39 = require('bip39');
+
+export interface WalletState {
+  seed: ArrayBuffer;
+}
+
+/**
+ * This class is responsible for all functions regarding
+ * the hierarchical deterministic wallet.
+ * The seed is created and verified here.
+ * The tokens are created and retrieved here.
+ */
+export class HDWallet {
+  private static readonly PREFIX: string = 'm';
+
+  private static readonly PURPOSE: string = '888';
+
+  private static readonly ACCOUNT: string = '0';
+
+  private static readonly SEPARATOR: string = '/';
+
+  /* cryptography manager: encrypts the seed with RSA key stored in IndexedDB */
+  private cryptoManager!: WalletCryptographyHandler;
+
+  /* local copy of encrypted seed */
+  private encryptedSeed!: ArrayBuffer;
+
+  /**
+   * this method initializes the wallet:
+   * 1. creating the crypto-manager (RSA key in IndexedDB).
+   * 2. encrypting the seed.
+   * 3. storing it in the application state through reducer.
+   * @param mnemonic the 12-word representation of the wallet seed
+   * @return true if the given mnemonic is valid, false otherwise
+   */
+  public async initialize(mnemonic: string): Promise<boolean> {
+    if (mnemonic === undefined) {
+      throw new Error('Error while initializing the wallet seed: seed is undefined');
+    }
+    const seedIsValid: boolean = bip39.validateMnemonic(mnemonic);
+    if (seedIsValid) {
+      this.cryptoManager = new WalletCryptographyHandler();
+      const seed: Uint8Array = await bip39.mnemonicToSeed(mnemonic);
+      await this.cryptoManager.initWalletStorage();
+      const encryptedSeed = await this
+        .encryptSeedToStoreInState(seed);
+      this.encryptedSeed = encryptedSeed;
+      WalletStore.store(encryptedSeed);
+    }
+    return seedIsValid;
+  }
+
+  /**
+   * retrieves and decrypts the wallet state (seed) from the application store.
+   * @return the plaintext wallet seed as a Uint8Array
+   */
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
Don't disable eslint whenever you have a problem, there's generally a very good reason for those remarks.
Google the error and the solution will be immediately obvious:
https://eslint.org/docs/rules/class-methods-use-this

Also, I suspect that even if you remove the eslint-disable, there won't be any linting issue here.